e2e_tests/aws_ec2/test_ec2_deployment.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import uuid
+import json
+
+from sklearn import datasets
+
+from e2e_tests.cli_operations import delete_deployment
+from e2e_tests.aws_ec2.utils import (
+    run_aws_ec2_create_command,
+    wait_for_instance_spawn,
+    send_test_data_to_endpoint,
+)
+
+logger = logging.getLogger('bentoml.test')
+
+
+def test_aws_ec2_deployment(iris_clf_service):
+    random_hash = uuid.uuid4().hex[:6]
+    deployment_name = f'tests-ec2-e2e-{random_hash}'
+    deployment_namespace = "test"
+    deployment_region = "ap-south-1"
```

Comment:
We should use the default region from `get_default_aws_region` in `yatai/deployment/aws_utils.py`

Reply:
fixed